src/frame-writer.hpp

```diff
@@ -11,7 +11,7 @@
 #include <atomic>
 #include "config.h"
 
-#define AUDIO_RATE 44100
+//#define AUDIO_RATE 44100
```

Comment:
You should remove items instead of commenting them, if they are not to be included in the final patch.

Reply:
That is my plan. I am still cleaning up.